src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -164,6 +165,20 @@
         <UseAbsolutePath Condition="'$(TestWithLocalLibraries)'=='true'">$([System.String]::Copy('%(_TestCopyLocalByFileNameWithoutDuplicates.SourcePath)').StartsWith('$(BinDir)'))</UseAbsolutePath>
       </IncludedFileForRunnerScript>
     </ItemGroup>
+    
+    <MakeDir Directories="$(OutputFolderForTestDependencies)" />
+    <Message Text="Generating $(OutputFolderForTestDependencies)/$(AssemblyName)-$(OSGroup).$(Platform).$(ConfigurationGroup).dependencylist.txt" />
+    <WriteLinesToFile
```

Comment:
Is there any real reason that assemblylist.txt and dependencylist.txt need to be different? Can we not use the same format and only output the one file?